.travis/package.sh

```diff
@@ -14,8 +14,13 @@ if [ "$TRAVIS_OS_NAME" = "linux" ] && [ "$PACKAGE" = "ON" ]; then
     mv ~/out/* .
 fi
 
+declare -A urls
+
 for file in SuperTux*; do
     echo "Uploading $file";
-    curl --upload-file "$file" "https://transfer.sh/$file"
-    echo
+    url=$(curl --upload-file "$file" "https://transfer.sh/$file")
```

Comment:
The view.php page will only show files from the last two weeks, so I can add a note, but they will drop off anyways